charmhelpers/contrib/openstack/files/check_openvswitch.py

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env python3
+# -*- coding: us-ascii -*-
```

Comment:
Not a big deal, but any particular reason why?

Reply:
This came from an hw-health check script that I used as a shell.  it can be removed.